R-package/LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2018
+COPYRIGHT HOLDER: Mouselimis Lampros <mouselimislampros@gmail.com>
```

Comment:
What is the license of the RGF R wrapper?

Reply:
The license is MIT. This appears in the [DESCRIPTION](https://github.com/mlampros/rgf-1/blob/master/R-package/DESCRIPTION) file (License: MIT + file LICENSE)

Reply:
OK, then please put the text of the MIT license here, because after opening `LICENSE` file everyone expects to see the license itself.

Reply:
I'm sorry but I can't do that. CRAN requires from R package maintainers to only mention the YEAR and the COPYRIGHT HOLDER. If I'll add the MIT text inside the LICENSE file then the package will not pass the tests and the package will not be accepted on CRAN. This is actually the reason that the DESCRIPTION file includes *License: MIT + file LICENSE*

Reply:
Are you sure in this? Seems that it's allowed to add test of a license. Please take a look:
https://github.com/dmlc/xgboost/blob/master/R-package/LICENSE

Reply:
This is an Apache license. I'm not familiar with this schema. However in case of the MIT license the [approved schema](https://cran.r-project.org/web/licenses/MIT) is the following : *... This is a template. Complete and ship as file LICENSE the following 2 lines (only) ...*